index.ts

```diff
@@ -101,7 +126,7 @@ async function main() {
           // - any label
           if (
             devpoolIssue.title !== projectIssue.title ||
-            devpoolIssue.state !== projectIssue.state ||
+            projectIssue.state == "close" ||
```

Comment:
I think we can keep the old code here `devpoolIssue.state !== projectIssue.state ||` and remove [this block](https://github.com/ubiquity/devpool-directory/blob/526277b8368f365ae787b0289f4a55503e7fed49/index.ts#L102C1-L113C12) totally, the logic should remain the same + we get less code

Reply:
Then it would execute in the case of project being open, devpool issue being closed even if the project issue has a assignee

Reply:
@rndquu

Reply:
Hey, shouldn't `projectIssue.state` to be `closed` instead of `close`?

Reply:
GOD I AM SO DUMB